stravalib/client.py

```diff
@@ -532,39 +585,39 @@ def get_friend_activities(self, limit=None):
 
     def create_activity(
         self,
-        name,
-        activity_type,
-        start_date_local,
-        elapsed_time,
-        description=None,
+        name:str,
```

Comment:
Here i started to add typing to the definitions. do we want to proceed with this approach? it will take a bit of time to do but could be worthwhile.

Reply:
Does this prevent us from adding types in the docstrings as well? If not, adding typing everywhere is an awful lot of work.

Reply:
@jsamoocha this is a good point. we could start by only adding docstring type info / cleaning up docstrings. 

Then, there are a few tools that can help automate the typing piece that i was going to look into such as:

https://github.com/Instagram/MonkeyType that was just shared with me today.  

perhaps that is a more efficient way to move forward?

Reply:
Yeah, let's try to do one thing at a time (i.e. add the typing in a later PR)

Reply:
great - i've removed the typing in there. better to keep each update simple.